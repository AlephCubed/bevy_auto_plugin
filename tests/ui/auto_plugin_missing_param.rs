use bevy_auto_plugin_macros::*;

#[auto_plugin]
fn plugin(_app: &bevy::app::App) {}

// dummy main
fn main() {
    
}