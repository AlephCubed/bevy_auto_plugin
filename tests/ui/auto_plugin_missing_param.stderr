error: auto_plugin requires attribute specifying the name of the `&mut bevy::app::App` parameter. Example: #[auto_plugin(app=app)]
 --> tests/ui/auto_plugin_missing_param.rs:3:1
  |
3 | #[auto_plugin]
  | ^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `auto_plugin` (in Nightly builds, run with -Z macro-backtrace for more info)
